### PR TITLE
Minor refactoring and reuse of already created temp directory

### PR DESCRIPTION
**Pull request by vibe13**

### CODE REVIEW

**Comment by rnc** [comment]
retest this please

**Comment by rnc** [comment]
retest this please

File: src/main/java/org/goots/exploder/Exploder.java [hunk 1]
```diff
@@ -196,7 +197,7 @@ public void unpack( ExploderFileProcessor processor, URL url ) throws InternalEx
                 useTemporaryDirectory();
             }
 
-            File target = new File ( Files.createTempDirectory( "exploder-" + UUID.randomUUID().toString() ).toFile(),
+            File target = new File ( targetDirectory,
                                      url.getFile().substring( url.getFile().lastIndexOf( '/' ) ) );
```
[the remark this answers is not in the document]
Comment (vibe13) [reply src/main/java/org/goots/exploder/Exploder.java]: ah ok right! ignore this then :-)

File: src/main/java/org/goots/exploder/Exploder.java [hunk 2]
```diff
@@ -46,6 +45,8 @@
      */
     public final static String ARCHIVE_UNPACK_SUFFIX = "-unpacked-" + UUID.randomUUID().toString();
 
+    public static final String ARCHIVE_EXTRACT_SUFFIX =  "exploder-" + UUID.randomUUID().toString();
+
     private final Logger logger = LoggerFactory.getLogger( getClass() );
```
[the remark this answers is not in the document]
Comment (vibe13) [reply src/main/java/org/goots/exploder/Exploder.java]: that would definitely make sense, +1000

**Comment by rnc** [comment]
After discussing this with @vibe13 we're closing this for now.

**Pull request closed without merging**